### ISSUE TITLE
Unknown column access

### ISSUE THREAD

**Issue by SabaunT** [polkascan/polkascan-pre#14]
After following build instructions I face errors like `Unknown column 'data_block.<field_name> in 'field list'` (whch is python exception message thrown by `sqlalchemy.exc.ProgrammingError`)  when worker tries to execute task.

Also it seems that there are different table schemes for "Kusama" and "dev" polkascan version:
 - when running Kusama, worker can't access field 'count_accounts': error raises with message `Unknown column 'data_block.count_accounts' in 'field list'`;

- when running dev, worker can't access field 'count_extrinsics_unsigned'. However, this field is present in `kusama_mysql_1` image.

**Comment by arjanz** [comment]
I believe you encountered a race condition during first initialisation, sometimes that happens if the database migrations are running during first run and the db is not in a ready state yet. Have you tried to 'docker-compose down' and then 'docker-compose up' again?

**Comment by SabaunT** [comment]
Alright
First of all, thanks for the polkascan - awesome job.
I tried to build and run polkascan again with 'master' version and 'v0.4.0' stable version and here is what happened.

'master':
1. I runned mysql initialization with `docker-compose -p dev -f docker-compose.yml up -d mysql`
2. after container was created I entered mysql in it with `docker exec -it dev_mysql_1 /bin/bash`, then `mysql -u root -p` with password "root", then I asked to view schema with `show columns from data_block;` and here is what I got:
```
+------------------+-------------+------+-----+---------+-------+
| Field            | Type        | Null | Key | Default | Extra |
+------------------+-------------+------+-----+---------+-------+
| id               | int         | NO   | PRI | NULL    |       |
| parent_id        | int         | NO   |     | NULL    |       |
| hash             | varchar(66) | NO   | UNI | NULL    |       |
| parent_hash      | varchar(66) | NO   | MUL | NULL    |       |
| state_root       | varchar(66) | NO   |     | NULL    |       |
| extrinsics_root  | varchar(66) | NO   |     | NULL    |       |
| count_extrinsics | int         | NO   |     | NULL    |       |
| count_events     | int         | NO   |     | NULL    |       |
| spec_version_id  | varchar(64) | NO   |     | NULL    |       |
| debug_info       | json        | YES  |     | NULL    |       |
+------------------+-------------+------+-----+---------+-------+
```

3. I run `docker-compose -p dev -f docker-compose.yml up --build` without stoping mysql container.
4. Everything successfully created and upped. However, celery worker while executing task raises an exception: `sqlalchemy.exc.ProgrammingError: (mysql.connector.errors.ProgrammingError) 1054 (42S22): Unknown column 'data_block.count_extrinsics_unsigned' in 'field list'`

If I down all the containers and up them again, I get an error `sqlalchemy.exc.InterfaceError: (mysql.connector.errors.InterfaceError) 2003: Can't connect to MySQL server on 'mysql:3306' (111 Connection refused)`. I suppose that is because after `docker-compose down` I should have upped mysql first and then all the other containers.


'v.0.4.0':
1. `docker-compose -p dev -f docker-compose.dev.yml up --build` fails build. This was discussed with Emiel in matrix chat. Long story short, problem is with ***requirements.txt*** file of harvester (and explorer-api), because they pull from github "master" versions of `py-scale-codec` and other github dependencies, although submodules commit-has version of these libs should be pulled.

Emiel told me that afer 0.5.x release such things will be fixed.

**Comment by emielvanderhoek** [comment]
We have released version: v0.5.0 with new build instruction in the README.md.
https://github.com/polkascan/polkascan-pre/releases/tag/v0.5.0
